rdf_differ/skos_history_wrapper.py

```diff
@@ -0,0 +1,125 @@
+"""
+skos_history_wrapper.py
+Date: 06/07/2020
+Author: Mihai Coșleț
+Email: coslet.mihai@gmail.com
+"""
+from pathlib import Path
+from shutil import copy
+from urllib.parse import urljoin
+
+from rdflib.util import guess_format
+
+from rdf_differ.config import get_envs
+from utils.file_utils import INPUT_MIME_TYPES, dir_exists, dir_is_empty
+
+
+class SKOSHistoryFolderSetUp:
+    def __init__(self, dataset: str, filename: str, old_version: str, new_version: str, root_path: str,
+                 version_1_name: str = 'v1', version_2_name: str = 'v2'):
+        self.dataset = dataset
+        self.filename = filename
+        self.old_version = old_version
+        self.new_version = new_version
+        self.root_path = root_path
+        self.version_1_name = version_1_name
+        self.version_2_name = version_2_name
+
+        self._check_root_path()
+
+    def generate(self):
+        v1 = Path(self.root_path) / self.dataset / 'data' / self.version_1_name
+        v2 = Path(self.root_path) / self.dataset / 'data' / self.version_2_name
+        v1.mkdir(parents=True)
+        v2.mkdir(parents=True)
+
+        copy(self.old_version, v1 / self.filename)
```

Comment:
perhaps it would be a better idea to use data-streams, but since now we're working with the `skos-history` bash file, it requires files, so I build the class around these requirements

Reply:
I recommend to leave it like this and make adjustments to the code when need arrises